config/locales/en.yml

```diff
@@ -138,6 +138,12 @@ en:
       save_contact_button: Save contact
     update:
       success: Contact has been updated successfully
+    destroy:
+      success: Contact has been deleted successfully
+    confirm_destroy:
+      title:
+        Are you sure you want to delete %{contact_name}?
+      guidance: This will remove the contact for %{contact_title} called %{contact_name} from the contacts list.
```

Comment:
This language doesn't feel good in my brain. Happy for it to ship now and we revisit it later with a content designer!

Reply:
Agreed!